Neolution.DotNet.Console.SampleAsync/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+WORKDIR /app
+
+# Copy everything
+COPY . ./
```

Comment:
This is a bit suboptimal. If anything changes in the parent folder (even if not relevant for this console app) the cache will be invalid. Also it would be better to first copy only the csproj file and restore from that. So if nothing changes in the csproj file (which is very likely) the cache will stay intact for that layer.

```
COPY ["Neolution.DotNet.Console.SampleAsync/Neolution.DotNet.Console.SampleAsync.csproj", "Neolution.DotNet.Console.SampleAsync/packages.lock.json", "Neolution.DotNet.Console.SampleAsync/"]

RUN dotnet restore

RUN dotnet publish -c Release -o out
```